build.dev.js

```diff
@@ -0,0 +1,430 @@
+"use strict";
```

Comment:
Yeah, thats going to be part of the refactor for sure, I need to clean up build.lib.js.